catalog-rest-service/src/main/java/org/openmetadata/catalog/EnumInterface.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.openmetadata.catalog;
+
+/** Interface which could be implemented by Enums classes */
+public interface EnumInterface {
```

Comment:
@nahuelverdugo do we need this. So far use of enums you can get a value directly without defining the interface like this no?

Reply:
Since we have different Enum types for `serviceType` on each service entity, we need this `EnumInterface` for the [ServiceEntityInterface.java](https://github.com/open-metadata/OpenMetadata/pull/6584/files#diff-3eadaa2b07b40826f43b90515affba217fef531c8cb5fc9c198356834e17442d).